src/main/webapp/app/modules/login/logout.tsx

```diff
@@ -12,7 +12,8 @@ export interface ILogoutProps extends StateProps, DispatchProps {}
 export class Logout extends React.Component<ILogoutProps> {
   componentDidMount() {
     this.props.reset();
-    this.props.resetProviderFilters();
+    this.props.resetProviderFilters(true);
```

Comment:
Why the filters are divided for map view and grid view. I think it should be the same.